Assets/Controllers/CameraMotionHandler.cs

```diff
@@ -63,24 +67,43 @@ void CheckIfCameraMoved()
         }
     }
 
-    public void MoveCamera(int upDown, int leftRight)
+    public void SetMoveTargetPosition(int upDown, int leftRight)
     {
-        float newX = transform.position.x + (leftRight * Time.deltaTime * cameraMoveSpeed);
-        float newZ = transform.position.z + (upDown * Time.deltaTime * cameraMoveSpeed);
+        float newX = transform.position.x + (leftRight * Time.deltaTime);
```

Comment:
Should this be a separate method that you pass the `transform.position` variable into and returns X and Z? It feels like this method is doing too much.